### DISCUSSION TITLE
Possible to create items within a browser field?

### DISCUSSION THREAD

**Discussion by sebastianlenton** [area17/twill#996] [Ideas]
Hi,

When choosing to link content via a browser field, is it possible to enable an option to create new records via the popup?

Eg, when I'm in a Project, the popup list of tags that appears when clicking "Add project tag" would also allow me to create new ones.

Thanks!

**Comment by ifox** [comment]
Hi Seb,

it should be, but I just checked and there are some issues, so let me get back to you later this week about this!

**Comment by adam-jones-net** [comment]
I've been looking for this feature as well.

**Comment by liamjcooper** [comment]
I am interested in this coming from a fresh project, is it being actively worked on? I can provide help where necessary.

**Comment by ConstantMath** [comment]
It would be amazing to have this feature, I can also help if needed. 
Amicalement.

**Comment by ifox** [comment]
This is being addressed in #731.
